docs/03_server/10_integration/04_gateways-and-streamers/02_fix-gateway.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 'FIX Gateways and Streamers - FIX Gateway'
+sidebar_label: 'FIX Gateway'
+id: fix-xlator
```

Comment:
you need to change the ID for this page, once it is the same id as the 05_fix-xlator.md file.